deploy/helm/sumologic/templates/metrics/remote-write-proxy/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.sumologic.metrics.remoteWriteProxy.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "sumologic.metadata.name.remoteWriteProxy.deployment" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.remoteWriteProxy.deployment" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "sumologic.labels.app.remoteWriteProxy.pod" . }}
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: {{ template "sumologic.labels.app.remoteWriteProxy.pod" . }}
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
```

Comment:
I would prefer to have `image`, `resources`,  `livenessProbe` and `readinessProbe`  configurable.
I think that it will be better to set default image for `nginx` with tag.